src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -160,15 +162,24 @@ const OptionRowLHN = (props) => {
                                 )
                             }
                             <View style={contentContainerStyles}>
-                                <DisplayNames
-                                    accessibilityLabel="Chat user display names"
-                                    fullTitle={optionItem.text}
-                                    displayNamesWithTooltips={displayNamesWithTooltips}
-                                    tooltipEnabled
-                                    numberOfLines={1}
-                                    textStyles={displayNameStyle}
-                                    shouldUseFullTitle={optionItem.isChatRoom || optionItem.isPolicyExpenseChat}
-                                />
+                                <View style={[styles.flexRow, styles.alignItemsCenter, styles.mw100, styles.overflowHidden]}>
+                                    <DisplayNames
+                                        accessibilityLabel="Chat user display names"
+                                        fullTitle={optionItem.text}
+                                        displayNamesWithTooltips={displayNamesWithTooltips}
+                                        tooltipEnabled
+                                        numberOfLines={1}
+                                        textStyles={displayNameStyle}
+                                        shouldUseFullTitle={optionItem.isChatRoom || optionItem.isPolicyExpenseChat}
+                                    />
+                                    {optionItem.isChatRoom && (
+                                        <TextPill
+                                            style={textPillStyle}
+                                            accessibilityLabel="Chat room name"
```

Comment:
@marcochavezf @shawnborton Can one of you confirm if this is the correct `accessibilityLabel` ?

Reply:
Deferring to @marcochavezf as I am unfamiliar with this one.

Reply:
Ah good question, since this only be used for Workspace name atm let's change the value to `Workspace name`

Reply:
Got it.

Reply:
Done